src/Tribe/Attendees.php

```diff
@@ -151,16 +151,25 @@ public function print_checkedin_totals() {
 	 * Returns the full URL to the attendees report page.
 	 *
 	 * @since 4.6.2
+	 * @since TBD - Added $post_id parameter to allow for conversion of provisional ID's into regular Post ID's.
 	 *
 	 * @param WP_Post $post
+	 * @param int $post_id The Post ID of the event.
 	 *
 	 * @return string
 	 */
 	public function get_report_link( $post ) {
+		// Convert provisional ID's into regular Post ID's.
+		if ( class_exists( '\TEC\Events\Custom_Tables\V1\Models\Occurrence', false ) ) {
+			$post_id = \TEC\Events\Custom_Tables\V1\Models\Occurrence::normalize_id( $post->ID );
+		} else {
+			$post_id = $post->ID;
+		}
+
```

Comment:
@redscar the filter doesn't normalize the ids for me. It's still returns the provisional ID when applied.

Reply:
Definitely odd, I had the hook working in another method in that class. https://github.com/the-events-calendar/event-tickets/blob/release/G23.greenpeas/src/Tribe/Attendees.php#L619

I saw your work with the helper function, I think that will work and if we need to add the hook into it in the future it would allow us to.